test/lib/mayaUsd/fileio/testCacheToUsd.py

```diff
@@ -471,8 +471,6 @@ def testEditAndMergeRigMayaRef(self):
         value = xformOp.Get()
         self.assertIn("xformOp:translate", value)
         self.assertIn("xformOp:rotateXYZ", value)
-        self.assertIn("xformOp:translate:channel1", value)
-        self.assertIn("xformOp:rotateXYZ:channel1", value)
```

Comment:
Removed these two lines because those are not valid any more, but I don't know if there's any other ways to test it...